### PR TITLE
[Test Structure] Fix markdown typo

### PR DESCRIPTION
**Pull request by MichaelCherry**
# Description

Adds an additional newline to the overview markdown so it is formatted properly
Fix for bug mentioned in #417

**PLEASE SEE THE TESTING PORTION BELOW**

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

## Screenshot
image

image

# Testing

Rollback to migration 13, then re migrate to 14
1.  open powershell, navigate to the repo
2.  `docker exec -it project-thundercat_backend_1 /bin/bash`
3.  `./manage.py migrate custom_models 0013_upload_background_tree_view_info`
4.  `./manage.py migrate custom_models`

# Checklist

Applicable for all code changes.

~~- [ ] I have commented my code, particularly in hard-to-understand areas~~
- [x] My changes generate no new compiler warnings
- [x] My changes generate no new accessibility errors and/or warnings
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
~~- [ ] I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)~~
- [x] My changes look good on IE 11+ and Chrome

### CODE REVIEW

**Comment by caleybrock** [comment]
Will this break the dev server if we don't do something special?

File: backend/tests/test_retrieve_test_view.py [hunk 1]
```diff
@@ -20,15 +20,15 @@ def test_get_real_sample(self):
             "is_public": True,
             "default_time": None,
             "test_type": "emib",
-            "meta_text": {
+             "meta_text": {
```
Comment (caleybrock) [inline backend/tests/test_retrieve_test_view.py]: is this extra space on purpose?

Comment (MichaelCherry) [reply backend/tests/test_retrieve_test_view.py]: dropped in 43cf3d0

**Comment by caleybrock** [comment]
The server won't break, but it won't have the new line until we rollback the migrations and rerun it.

**Pull request merged**